packages/atomic/src/pages/examples/suggestions.html

```diff
@@ -82,7 +82,9 @@
         <atomic-layout-section section="search">
           <atomic-search-box suggestion-timeout="1000">
             <custom-suggestions></custom-suggestions>
-            <atomic-search-box-query-suggestions></atomic-search-box-query-suggestions>
+            <atomic-search-box-query-suggestions
+              icon="../../assets/magnifier.svg"
```

Comment:
using another icon from the assets we already have could be better than adding new ones in pages/magnifier

> Use a value that starts with `assets://`, to display an icon from the Atomic package.

Reply:
I can't seem to make it work - setting it to `assets://search.svg` for example, seems to do something
![Screenshot 2022-05-13 at 10 03 51](https://user-images.githubusercontent.com/30511433/168250134-9013e4d9-aa91-4f5c-a3f5-e5d7acec7c1b.png)
which is different from any other string:
![Screenshot 2022-05-13 at 10 03 11](https://user-images.githubusercontent.com/30511433/168250017-4bddaf4c-7d9d-4abc-9700-bc009f53c138.png)
but still doesn't load the icon, as you can see! do you know why?

Reply:
Oh let me look...

Reply:
ermmm that's embarrassing! sorry 😬  I tried a couple and must have been unlucky to choose only white ones!!